### PR TITLE
added servlet and jsp and edited web.xml for profile pages prototype

### PR DESCRIPTION
**Pull request by asma526**
Created the simple prototype by making a servlet and jsp for the profile pages. Also edited the web.xml to add the /prototype url.

### CODE REVIEW

File: src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -66,4 +66,14 @@
...
   </servlet-mapping>
 
+  <servlet>
+    <servlet-name>ProfilePagesServlet</servlet-name>
+    <servlet-class>codeu.controller.ProfilePagesServlet</servlet-class>
+  </servlet>
+
+  <servlet-mapping>
+    <servlet-name>ProfilePagesServlet</servlet-name>
+    <url-pattern>/profilepages</url-pattern>
```
Comment (erschwartz) [inline src/main/webapp/WEB-INF/web.xml]: I'd add a - between profile and pages

File: src/main/webapp/WEB-INF/view/profile-pages.jsp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<html>
+<head>
+  <title>Profile Pages</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/profilepages">Profile Pages</a>
```
Comment (erschwartz) [inline src/main/webapp/WEB-INF/view/profile-pages.jsp]: /profile-pages according to other comment

File: src/main/webapp/WEB-INF/view/profile-pages.jsp [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else{ %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+  </nav>
+
```
Comment (erschwartz) [inline src/main/webapp/WEB-INF/view/profile-pages.jsp:20]: Please write a TO-DO here in format
`TODO($USERNAME): Take out the trash`

**Review by erschwartz — CHANGES_REQUESTED** [review]
Good start, Asma!

**Review by ilianac — APPROVED** [review]

**Comment by erschwartz** [comment]
LGTM

**Review by erschwartz — APPROVED** [review]

**Pull request merged**